pkg/controller/spinnakervalidating/spinnakervalidating_controller.go

```diff
@@ -72,7 +194,23 @@ func (v *spinnakerValidatingController) InjectClient(c client.Client) error {
 }
 
 // InjectDecoder injects the decoder.
-func (v *spinnakerValidatingController) InjectDecoder(d admission.Decoder) error {
+func (v *spinnakerValidatingController) InjectDecoder(d *admission.Decoder) error {
 	v.decoder = d
 	return nil
 }
+
+func (v *spinnakerValidatingController) collectErrors(results []validate.ValidationResult) error {
```

Comment:
We're losing errors by just collecting the message. I'd think `ValidationResult` can return a `validationError` that can wrap one or more errors. At the end of parallel validations, we can `ValidationResult.merge([]ValidationResult)` and keep the wrapped errors. We have at least 2 parallel validations: all providers/ci/... and for each provider parallel for each account.